bin/cli.js

```diff
@@ -1,17 +1,248 @@
 #!/usr/bin/env node
 'use strict';
 
-const metasync = require('metasync');
+const program = require('commander');
+const common = require('metarhia-common');
+const karma = require('karma');
+const path = require('path');
+const fs = require('fs');
 
-const walk = require('../lib/walk');
+const TEST_DIR = 'test';
 
-console.log('Metatest CLI Stub');
+const splitOpts = v => v.split(',');
+
+const cliOptions = [
+  [ '--node',                    'Run in node.js environment' ],
+  [ '--browser',                 'Run in browser environment' ],
+  [ '--browser-only <items>',    'Browser only tests patterns', splitOpts, [] ],
```

Comment:
Is it possible to provide multiple patterns via single option?
(like `--node-only 'abc' 'xyz' '123'`)
If it's not the please change description to `--browser-only <pattern>` and `Browser only tests pattern` here and in the other places.